tensorflow/python/kernel_tests/cwise_ops_test.py

```diff
@@ -55,6 +56,30 @@
 _XOR = lambda x, y: x ^ y
 _INV = lambda x: ~x
 
+# @ operator supported since python 3.5.
+if sys.version_info >= (3, 5):
+  # Use eval() to avoid a compilation error on earlier versions.
+  _MATMUL = lambda x, y: eval("x @ y")
+else:
+  # For earlier versions of python, emulate regular behavior.
+  # Useful to build and test for 3.5+ on earlier versions.
+  def _MATMUL(x, y):
```

Comment:
Actually, after reading further, it's reasonable to test that `__matmul__` works as expected in lower versions of Python.  So keep `_MATMUL`, but still split out into a separate test file.

Alternately, you could put it in `matmul_op_test.py`.